.github/workflows/release-ninja-binaries.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         mkdir build && cd build
         cmake -DCMAKE_BUILD_TYPE=Release ..
         cmake --build . --parallel --config Release --target ninja
+        cmake --build . --parallel --config Release --target ninja_test
```

Comment:
Why not remove `--target ninja` from the previous command? Should also be faster.

Reply:
Alright.